pkg/metric/metric.go

```diff
@@ -0,0 +1,25 @@
+package metric
```

Comment:
Another possible library that does something similar: https://github.com/armon/go-metrics ... not saying we should use this in this case, but it came to mind as there are several different "kinds" of metrics that may need to be handled and our own library may get complicated over time.

Reply:
The only reason I went ahead and implemented YET another "standard" is purely to gain more control over the metrics and what we do with them.

I'm not against replacing it. Surely I'd be happier if we could avoid complex implementation in the future.

We've decided to roll without it for the `alpha` and perhaps revisit in the future.